### ISSUE TITLE
请求优化

### ISSUE THREAD

**Issue by hzfei** [QwenLM/Qwen#791]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

### 备注 | Anything else?

![1](https://github.com/QwenLM/Qwen/assets/42926130/24af9515-d1be-46c1-8979-b9ef3bc0e912)
![2](https://github.com/QwenLM/Qwen/assets/42926130/c072ea8c-28a2-45fa-a6d6-d6481193f384)

**Comment by jklj077** [comment]
In the context of fact-based, memory-driven Q&A tasks, the scale of the employed model plays a substantial role in determining the level of accuracy achieved. Should your current implementation utilize a more modestly sized model, transitioning to a larger, more comprehensive model could potentially enhance performance. An upgrade to Qwen1.5 is among the options worth exploring. It is essential to underscore that with generative models, there is inherently no guarantee regarding the absolute accuracy or exhaustiveness of the produced content.
